README.md

```diff
@@ -31,10 +31,26 @@ Please cite the following paper if you use ensmallen in your research and/or
 software. Citations are useful for the continued development and maintenance of
 the library.
 
-* S. Bhardwaj, R. Curtin, M. Edel, Y. Mentekidis, C. Sanderson.
-  [ensmallen: a flexible C++ library for efficient function optimization](http://www.ensmallen.org/files/ensmallen_2018.pdf).
-  Workshop on Systems for ML and Open Source Software at NIPS 2018.
-
```

Comment:
Maybe would it be worth leaving both the "clean" version as well as the BibTeX source?

Reply:
Totally agree,